functions_build.sh

```diff
@@ -199,7 +199,8 @@ cleanup_install_dir() {
                         execute strip lib/lib*.a
                         # On CentOS 5, libffi and OpenSSL install to lib64/.
                         if [ -d lib64 ]; then
-                            execute strip lib64/lib*.a
+                            echo "lib64/ found!"
+                            exit 88
```

Comment:
It now exits on existence of the `lib64` dir? Does it no longer need to work on CentOS 5?

Reply:
Solved with https://github.com/chevah/pythia/pull/34/commits/cc4eeab627a4fb975f61d281ddbaca837fd695d3.